### PR TITLE
sys/riotboot: uncrustify

### PR DESCRIPTION
**Pull request by fjmolinas**
### Contribution description

While looking at #15912 I realized riotboot files were full of uncrustify issues. To ease reviewing I prefered to split uncrustify changes out and add it to the whitelist.

### Testing procedure

- green Murdock
- binaries should not have changed

### Issues/PRs references

Waiting for #15912

### CODE REVIEW

File: cpu/stm32/include/cpu_conf.h [hunk 1]
```diff
@@ -116,6 +116,11 @@ extern "C" {
 #define FLASHPAGE_SIZE                  (128U)
 #endif
 
+#if defined(CPU_FAM_STM32L0) || defined(CPU_FAM_STM32L1) || \
```
Comment (kaspar030) [inline cpu/stm32/include/cpu_conf.h:119]: I think this PR still includes the other commits.

Comment (fjmolinas) [reply cpu/stm32/include/cpu_conf.h:119]: It does. that is why its waiting for the other PR, since it will have a conflict anyway

Comment (fjmolinas) [reply cpu/stm32/include/cpu_conf.h]: Ah I understood what you meant, that happened in the original PR, so they are removed now. Thanks for pointing this out.

**Comment by fjmolinas** [comment]
All green here @kaspar030

**Review by kaspar030 — APPROVED** [review]
ACK.

**Pull request merged**